ashlar/serializer_fields.py

```diff
@@ -27,6 +27,49 @@ def to_internal_value(self, value):
             raise ValidationError('Array or object required')
 
 
+class DropJsonKeyException(Exception):
+    pass
```

Comment:
This isn't an error condition; it's just a way for the transform function to signal that the key under consideration should be dropped. I also considered having the transform function return `None` to drop a field, but that could get confusing if you actually wanted to insert `null` into a JSON response for some reason, so I went with the exception route.